src/utils/process.rs

```diff
@@ -872,10 +891,15 @@ pub mod tests {
     #[test]
     fn test_process_blame_no_parent_found() {
         let two_trees = MockProcInfo::with(&[
-            (2, 100, "-shell", None),
-            (3, 100, "git blame src/main.rs", Some(2)),
-            (4, 100, "call_delta.sh", None),
-            (5, 100, "delta", Some(4)),
+            (Pid::from_u32(2), 100, "-shell", None),
+            (
+                Pid::from_u32(3),
```

Comment:
What about moving all the `Pid::from_u32(..)` calls into `MockProcInfo::with(..)`?

Reply:
What do you think @dandavison ?

Reply:
Yes, I'd also prefer to continue to use native integers where possible, and only wrap as `sysinfo::Pid` where necessary -- i.e. close to the library call boundaries.

Reply:
Fine by me! Will sent an update as soon as possible.